www/googlemaps-cdv-plugin.js

```diff
@@ -2632,10 +2632,13 @@ function getAllChildren(root) {
     var clickable;
     var style, displayCSS, opacityCSS, visibilityCSS, node, clickableSize;
 
-    var allClickableElements = Array.prototype.slice.call(root.querySelectorAll(':not([data-clickable="false"])'));
-    var clickableElements =  allClickableElements.filter(function(i) {return i != root;});
+    root = document.querySelector('.main-region.foreground');
```

Comment:
here's the "cheat" https://github.com/cabify/cabify_js/blob/master/mobile/source/javascripts/app/views/map/native_gmaps.js.coffee#L4